src/containers/form/PhotoBlock.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import './PhotoForm.sass'
+
+class PhotoBlock extends Component {
+    constructor() {
+        super();
+
+        this.fileValidation = this.fileValidation.bind(this);
+    }
+    fileValidation(event) {
+        let self = this;
+        let allowedExtensions = /(\.jpg|\.jpeg|\.png)$/i;
+        let filePath = event.target.value;
+        let file = event.target.files[0];
+        let size;
+
+        event.persist();
+
+        if (file){
+            size = file.size;
+            if(!allowedExtensions.exec(filePath)){
+                this.validate(false, 'Incorrect file extension');
+                event.target.value = '';
+            } else {
+                let img = new Image();
+
+                if (size >= 40000 && size <= 5000000 ) {
+                    let reader = new FileReader();
+
+                    reader.onload = (e) => {
+                        let imgLink = e.target.result;
+                        img.src = imgLink;
+
+                        img.onload = function() {
+                            let width = this.width;
+                            let height = this.height;
+
+                            if (width >= 200 && height >= 200) {
+                                self.props.addPhotoFile(imgLink);
+                            } else {
+                                self.validate(false, 'Width and Height must be 200px or more');
+                                event.target.value = '';
+                            }
+                        };
+                    };
+                    reader.readAsDataURL(file);
+                } else {
+                    this.validate(false, `File size should be from 40000 to 5000000 bytes. Current is: ${size}`);
```

Comment:
this is not very user-friendly. I don't believe that all users know how many bytes inside 1MB